src/placeos-models/user.cr

```diff
@@ -161,24 +165,20 @@ module PlaceOS::Model
       User.find_all([auth_id], index: :authority_id)
     end
 
-    secondary_index :email
+    secondary_index :email_digest
```

Comment:
this isn't required as ensure unique creates the index

Reply:
Just thought I’d be explicit.